### DISCUSSION TITLE
Combining Categories (soft and strict)

### DISCUSSION THREAD

**Discussion by HitArrowLegend** [wintercms/wn-blog-plugin#9] [Ideas]
Hello, 

I already posted this idea for rainlab blog almost a year ago and although I still think it's a valid suggestion, I understand that this may not be useful to everyone, although I don't believe this would break anything for anyone already using the plugin.

(At the time, I decided to use a different plugin but today, things are different.)

https://github.com/rainlab/blog-plugin/issues/500

The reason why I think that at least idea A should be added in core is that it's possible to exclude a list of multiple categories. Having the same option for including categories only makes sense. (The strictness of the inclusion, or Idea B, is more a personnal need and therefore is less important, although it's a nice feature for a blog plugin imo).

**Comment by bennothommo** [comment]
I like this idea (or at the very least, some way to bulk migrate posts between categories). We'll look into it at some point.

**Reply by HitArrowLegend** [reply]
Im not sure this is what i was proposing but I also like this bulk migrating idea
